cloudplatform/cloudplatform-core-scp-cf/src/test/java/com/sap/cloud/sdk/cloudplatform/ScpCfCloudPlatformTest.java

```diff
@@ -645,4 +649,35 @@ public void underlyingPropertiesWillBeOverwrittenInServiceBinding()
         assertThat(parsedBinding.get("tags").getAsJsonArray()).isEmpty();
         assertThat(parsedBinding.get("credentials").getAsJsonObject().asMap()).isEmpty();
     }
+
+    @Test
+    public void dwcBindingDoesNotThrow()
+    {
+        final MegacliteServiceBinding serviceBinding =
+            MegacliteServiceBinding
+                .forService(ServiceIdentifier.DESTINATION)
+                .providerConfiguration()
+                .name("destination-paas")
+                .version("v1")
+                .build();
+
+        // sanity check
+        assertThat(serviceBinding.getName()).isEmpty();
+        assertThat(serviceBinding.getServiceName()).hasValue(String.valueOf(ServiceIdentifier.DESTINATION));
+        assertThat(serviceBinding.getServicePlan()).isEmpty();
+        assertThat(serviceBinding.getTags()).isEmpty();
+        // The dwcConfiguration cannot find the DWC_APPLICATION env var, so the credentials are empty
+        assertThat(serviceBinding.getCredentials()).isEmpty();
+        assertThat(serviceBinding.getKeys()).isEmpty();
+
+        // setup subject under test
+        final ScpCfCloudPlatform sut = new ScpCfCloudPlatform();
+        sut.setServiceBindingAccessor(() -> Collections.singletonList(serviceBinding));
+
+        // does not throw
+        final Map<String, JsonArray> parsedVcapServices = sut.getVcapServices();
+
+        // assert
+        assertThat(parsedVcapServices).containsOnlyKeys(String.valueOf(ServiceIdentifier.DESTINATION));
```

Comment:
[minor]
`String.valueOf` seems odd here...

```suggestion
        assertThat(parsedVcapServices).containsOnlyKeys(ServiceIdentifier.DESTINATION.toString());
```